app/views/sections/_feedback-prompt.html.erb

```diff
@@ -0,0 +1,18 @@
+<div data-controller="feedback-prompt" role="complementary">
+  <div class="dialog" data-target="feedback-prompt.dialog">
+    <div class="dialog__background"></div>
+    <div class="dialog__content">
+      <h3>We'd like your feedback</h3>
+      <p>
+        To help us improve the Get Into Teaching website, we'd like to ask you a few short questions about your experience today.
+      </p>
+      <a href="https://docs.google.com/forms/d/e/1FAIpQLSfIBfaYUk07Fb6dxUKAokiPoGGL73ZYVGHM1mSNyA-K-6QRbA/viewform" data-action="feedback-prompt#disable" class="call-to-action-button">
+        Yes, I'll answer the questions <i class="fas fa-chevron-right"></i>
+      </a>
+      <br />
+      <a class="secondary-link" data-action="feedback-prompt#close #feedback-prompt#disable">
+        No thank you <i class="fas fa-chevron-right"></i>
```

Comment:
This needs a screen reader annotation to work out of context
"I don't want to answer the questions"